tuf/testutils/repo.go

```diff
@@ -43,23 +42,34 @@ func createKey(cs signed.CryptoService, gun, role string) (data.PublicKey, error
 // EmptyRepo creates an in memory key database, crypto service
```

Comment:
Should we remove the reference to the key database since we already mention a crypto service?